cohorts/file_helpers.py

```diff
@@ -46,40 +46,18 @@ def cohort_files(cohort_id, inc_filters=None, user=None, limit=25, page=1, offse
 
     user_email = user.email
     user_id = user.id
-
-    resp = None
     db = None
     cursor = None
-    query_limit = limit
-    type_conditions = ""
     limit_clause = ""
     offset_clause = ""
-
-    filter_counts = None
     file_list = []
     total_file_count = 0
 
-    case_barcode = None
-    case_barcode_condition = ''
-
-    # DICOM uses BQ, and that WHERE clause builder can handle the LIKE clause,
-    # but the MySQL WHERE clause builder can't
-    if not type == 'dicom':
-        if 'case_barcode' in inc_filters:
-            case_barcode = inc_filters['case_barcode']
-            del inc_filters['case_barcode']
-
-        if case_barcode:
-            case_barcode_condition = "AND LOWER(cs.case_barcode) LIKE %s"
-            case_barcode = "%{}%".format(case_barcode)
-
```

Comment:
The addition of the %% around the user text doesn't seem to be added back in else-where--is it coming from the request now?